2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs

```diff
@@ -33,7 +33,17 @@ public static void CreatePublicClient()
             {
                 builder.WithBroker();
                 builder = builder.WithIosKeychainSecurityGroup("com.microsoft.adalcache");
-                builder = builder.WithRedirectUri(App.BrokerRedirectUriOnIos);
+                
+                switch (Device.RuntimePlatform)
```

Comment:
@jmprieur This switch will make it easier for customers to use the correct redirect URI when using broker because they are different for iOS and android